README.md

```diff
@@ -67,7 +68,7 @@ If you work at Cognite, you can find examples of code use in the repository ML-e
 Want to contribute? Check out [CONTRIBUTING](https://github.com/cognitedata/cognite-sdk-python/blob/master/CONTRIBUTING.md).
 
 ## Documentation
-* [SDK Documentation](http://cognite-sdk-python.readthedocs.io/en/latest/)
+* [SDK Documentation](https://cognite-cognite-python-sdk.readthedocs-hosted.com/en/latest/)
```

Comment:
double 'cognite'?

Reply:
yes.. I have sent them a mail because i couldn't figure out how to change the url. But at least ads are gone now :)